ecflow/ecf/scripts/gfs/atmos/post/jgfs_atmos_post_f001.ecf

```diff
@@ -4,7 +4,7 @@
 #BSUB -J %E%gfs_atmos_post_%FHR%_%CYC%
 #BSUB -o %COM%/output/%ENVIR%/today/gfs_atmos_post_%FHR%_%CYC%.o%J
 #BSUB -W 6:00
```

Comment:
6 hours for a single forecast file to process is too long?
Does the task in here take 6 hours?

There are 384 hours of forecasts to process.
Do we need 384 `jgfs_atmos_post_fHHH.ecf` files to maintain that?